src/dnc/azext_dnc/generated/commands.py

```diff
@@ -52,10 +55,11 @@ def load_command_table(self, _):
     with self.command_group('dnc delegated-subnet-service', dnc_delegated_subnet_service,
                             client_factory=cf_delegated_subnet_service) as g:
         g.custom_command('list', 'dnc_delegated_subnet_service_list')
-        g.custom_command('delete', 'dnc_delegated_subnet_service_delete', confirmation=True)
-        g.custom_command('patch-detail', 'dnc_delegated_subnet_service_patch_detail')
-        g.custom_command('put-detail', 'dnc_delegated_subnet_service_put_detail')
-        g.custom_command('show-detail', 'dnc_delegated_subnet_service_show_detail')
+        g.custom_show_command('show', 'dnc_delegated_subnet_service_show')
+        g.custom_command('create', 'dnc_delegated_subnet_service_create', supports_no_wait=True)
+        g.custom_command('delete', 'dnc_delegated_subnet_service_delete', supports_no_wait=True, confirmation=True)
+        g.custom_command('patch-detail', 'dnc_delegated_subnet_service_patch_detail', supports_no_wait=True)
```

Comment:
Please use "update" instead of "patch-detail"

Reply:
removed